common/cgroup/cgroup.go

```diff
@@ -124,27 +167,45 @@ func parseCgroups(f io.Reader) (map[int][]string, error) {
 	return cgroups, nil
 }
 
-// GetEnabledCgroups returns a map with the enabled cgroup controllers grouped by
+// GetEnabledLegacyCgroups returns a map with the enabled cgroup controllers grouped by
 // hierarchy
-func GetEnabledCgroups() (map[int][]string, error) {
+func GetEnabledLegacyCgroups() (map[int][]string, error) {
 	cgroupsFile, err := os.Open("/proc/cgroups")
 	if err != nil {
 		return nil, err
 	}
 	defer cgroupsFile.Close()
 
-	cgroups, err := parseCgroups(cgroupsFile)
+	cgroups, err := parseLegacyCgroups(cgroupsFile)
 	if err != nil {
 		return nil, errwrap.Wrap(errors.New("error parsing /proc/cgroups"), err)
 	}
 
 	return cgroups, nil
 }
 
-// GetControllerDirs takes a map with the enabled cgroup controllers grouped by
+// GetEnabledControllers returns a list of enabled cgroup controllers
+func GetEnabledControllers() ([]string, error) {
```

Comment:
This is v2 only. The way the functions are named tries to use "Legacy" for v1 and nothing for v2. We can change it to be specific and mention v1 and v2 in function names.